std_msgs/Header header
std_msgs/String cmd_type
std_msgs/String actuator_name
int16 position
int16 velocity